src/server.ts

```diff
@@ -0,0 +1,9 @@
+import express from 'express';
+import path from 'path';
+
+const app = express();
+
+app.use(express.static(path.join(__dirname, 'public')));
```

Comment:
you'll need bodyparser, helmet, cors as well, since this is going to be accessed via a website

Reply:
app.use(express.json());
I think is the body-parser now-a-days? it got moved after it was taken out and put back in again. (referencing :https://stackoverflow.com/questions/47232187/express-json-vs-bodyparser-json)

And cors and helmet incoming, configuration of those two I would like to be done by someone a bit more experienced in the back end - if possible:)